From b09959af72aef7ee937e7ed7acc81dc86557114f Mon Sep 17 00:00:00 2001
From: Mike Hommey <mh@glandium.org>
Subject: Revert "[instrprof] Fix issue in binary-ids-padding.test"

This reverts commit 1ec021467da65c06cd4aa24ce81898b0759bd16b, which is a
preliminary to revert 1ae7d83803e45f6053ec6a606f259653846926b8.

---
 llvm/lib/ProfileData/InstrProfReader.cpp | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/llvm/lib/ProfileData/InstrProfReader.cpp b/llvm/lib/ProfileData/InstrProfReader.cpp
index d0714c9b4665..4dfc3bab932d 100644
--- a/llvm/lib/ProfileData/InstrProfReader.cpp
+++ b/llvm/lib/ProfileData/InstrProfReader.cpp
@@ -662,13 +662,14 @@ template <class IntPtrT>
 Error RawInstrProfReader<IntPtrT>::readBinaryIds(
     std::vector<llvm::object::BuildID> &BinaryIds) {
   return readBinaryIdsInternal(*DataBuffer, BinaryIdsSize, BinaryIdsStart,
-                               BinaryIds, getDataEndianness());
+                               BinaryIds,
+                               llvm::support::endian::system_endianness());
 }
 
 template <class IntPtrT>
 Error RawInstrProfReader<IntPtrT>::printBinaryIds(raw_ostream &OS) {
   return printBinaryIdsInternal(OS, *DataBuffer, BinaryIdsSize, BinaryIdsStart,
-                                getDataEndianness());
+                                llvm::support::endian::system_endianness());
 }
 
 namespace llvm {
-- 
2.39.0.1.g6739ec1790

